test/system/252-quadlet.bats

```diff
@@ -723,10 +723,18 @@ EOF
 @test "quadlet - rootfs" {
     skip_if_no_selinux
     skip_if_rootless
```

Comment:
I am somewhat sure these can be removed now?

Reply:
Wouldn't the mounts need complicated `unshare`? Would those work inside quadlet?

Reply:
ah right yes you would need to use unshare rootless but yes as they all share the same userns the mounts should work with quadlet. Anyway I guess it is not really important to test it rootless. (although I do not get the no_selinux skip either)

I guess I just merge it as it is the same as before

/lgtm